### PR TITLE
[CPREQ-8832] Support Kubernetes v1.30.1

### PR DESCRIPTION
**Pull request by ilia1243**
### Description
* Support Kubernetes v1.30.1

### Test Cases

Install or upgrade cluster to Kubernetes v1.30.1

### Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Integration CI passed
- [ ] Unit tests. If Yes list of new/changed tests with brief description
- [x] There is no merge conflicts

### CODE REVIEW

File: kubemarine/procedures/backup.py [hunk 1]
```diff
@@ -410,7 +410,13 @@ def _handle(self, payload: ParserPayload) -> None:
         items_by_resource: Dict[str, List[str]] = {}
 
         def append_item(api_version: str, kind: str, item: str) -> None:
-            resource_name = next(r['name'] for r in resources if r['apiVersion'] == api_version and r['kind'] == kind)
+            resource_name = next(
+                r['name'] for r in resources
+                if r['apiVersion'] == api_version
+                and (r['kind'] == kind
+                     or kind in ('ValidatingAdmissionPolicyList', 'ValidatingAdmissionPolicyBindingList')
+                     and r['kind'] == kind.rstrip('List'))
```
Comment (ilia1243) [inline kubemarine/procedures/backup.py:418]: Not an appropriate usage of `rstrip`, to be fixed
https://stackoverflow.com/questions/34544247/understanding-pythons-lstrip-method-on-strings

**Review by vlar0816 — APPROVED** [review]

**Comment by pranavcracker** [comment]
As latest versions are available for calico, crictl and nginx-ingress-controller, can we add support for them?
We have added similar support in v1.29.4, v1.28.9 & v1.27.13
https://github.com/Netcracker/KubeMarine/pull/662

**Comment by ilia1243** [comment]
> As latest versions are available for calico, crictl and nginx-ingress-controller, can we add support for them? We have added similar support in v1.29.4, v1.28.9 & v1.27.13 PR #662

I will update compatibility map to the new plugins versions if PR 662 is merged first. Otherwise, I suggest to do that within PR 662.

**Comment by ilia1243** [comment]
> As latest versions are available for calico, crictl and nginx-ingress-controller, can we add support for them? We have added similar support in v1.29.4, v1.28.9 & v1.27.13 PR #662

I will update compatibility map to the new plugins versions if PR 662 is merged first. Otherwise, I suggest to do that within PR 662.

**Comment by koryaga** [comment]
> > As latest versions are available for calico, crictl and nginx-ingress-controller, can we add support for them? We have added similar support in v1.29.4, v1.28.9 & v1.27.13 PR #662
> 
> I will update compatibility map to the new plugins versions if PR #662 is merged first. Otherwise, I suggest to do that within PR 662.

pls proceed

**Pull request merged**